package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Remove this lock file, we use `pnpm` instead of traditional `npm`.
So it doesn't need this file.